librz/core/ctypes.c

```diff
@@ -305,20 +305,23 @@ static void core_types_struct_print(RzCore *core, RzBaseType *btype, RzOutputMod
 	switch (mode) {
 	case RZ_OUTPUT_MODE_JSON: {
 		rz_return_if_fail(pj);
+		if (!btype) {
+			break;
+		}
```

Comment:
```suggestion
```

Is this the same as the previous PR or another case?
Anyway, this is not necessary.

Reply:
This commit just could be dropped from the PR, it's easier.